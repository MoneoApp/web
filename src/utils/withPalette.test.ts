import { withPalette } from './withPalette';

it('should work without arguments', () => {
  expect(withPalette()?.styles).toMatchSnapshot();
});

it('should inject the correct css rules', () => {
  expect(withPalette(['grey-500', 'grey-0'])?.styles).toMatchSnapshot();
});
