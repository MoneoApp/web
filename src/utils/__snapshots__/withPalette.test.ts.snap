// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should inject the correct css rules 1`] = `"color:var(--grey-500);background-color:var(--grey-0);;label:withPalette;"`;

exports[`should work without arguments 1`] = `undefined`;
